config/config.exs

```diff
@@ -50,6 +50,8 @@ config :phoenix, :json_library, Jason
 # Disable rustler precompiled NIFs
 config :rustler_precompiled, :force_build, rhai_rustler: true
 
+config :wanda, :jwt_authentication, enabled: true
```

Comment:
Are we using the default expiration time?
I think it would be valuable to make it optional. It is 2 hours always otherwise

Reply:
we are not, expiration is just for generating jwt and wanda does not generate them just read them.